docs/usage.md

```diff
@@ -287,3 +288,104 @@ podman build . \
 ```
 
 Details: [write the Containerfile](#write-the-dockerfile-or-containerfile), [build the container](#build-the-container)
+
+
+## Example: cargo
+
+### Prefetch dependencies (cargo)
+
+```shell
+mkdir -p playground/pure-rust
+cd playground/pure-rust
+git clone git@github.com:sharkdp/bat.git --branch=v0.22.1
+cachi2 fetch-deps --source bat '{"type": "cargo"}'
+```
+
+### Generate environment variables (cargo)
+
+#### Alternative crates.io
+Cargo must be configured differently when crates.io will be replaced with an alternative registry to crates.io (or at least similar way - something that the [cargo plugin for nexus](https://github.com/sonatype-nexus-community/nexus-repository-cargo) seems to handle) or from a local path.
+
+For enabling an alternative crates.io registry the only required env is 
+
+```
+CARGO_REGISTRIES_MY_REGISTRY_INDEX=https://my-intranet:8080/git/index
+```
+
+[cargo docs on using an alternate registry](https://doc.rust-lang.org/cargo/reference/registries.html#using-an-alternate-registry)
+
+Unfortunately I'm still clueless on how to run an alternative registry (and this don't seem to align with cachi2 ethos), so I'm skipping this option.
```

Comment:
We'd prefer a completely offline install, yeah

Though we will eventually have to figure out how to store the downloaded dependencies permanently, either for the Cachito+OSBS integration or in the future for RHTAP builds